akka-http-core/src/test/scala/akka/http/impl/model/parser/HttpHeaderSpec.scala

```diff
@@ -141,12 +141,12 @@ class HttpHeaderSpec extends FreeSpec with Matchers {
       "Authorization: Fancy QWxhZGRpbjpvcGVuIHNlc2FtZQ==" =!=
         Authorization(GenericHttpCredentials("Fancy", "QWxhZGRpbjpvcGVuIHNlc2FtZQ=="))
       """Authorization: Fancy yes="n:o", nonce=42""" =!=
-        Authorization(GenericHttpCredentials("Fancy", Map("yes" → "n:o", "nonce" → "42"))).renderedTo(
-          """Fancy yes="n:o",nonce=42""")
-      """Authorization: Fancy yes=no,nonce="4\\2"""" =!=
-        Authorization(GenericHttpCredentials("Fancy", Map("yes" → "no", "nonce" → """4\2""")))
-      """Authorization: Other yes=no,empty=""""" =!=
-        Authorization(GenericHttpCredentials("Other", Map("yes" → "no", "empty" → "")))
+        Authorization(GenericHttpCredentials("Fancy", Map("nonce" → "42", "yes" → "n:o"))).renderedTo(
```

Comment:
I understand this change is because now using `TreeMap` the contents are sorted by key, right?

Reply:
Correct. I don't think we ever promised to keep the order of the keys (and still don't), so in that sense this test is too specific.